tests/base_test_mixins.py

```diff
@@ -136,7 +140,63 @@
 ]
 
 
-class SuperuserCreateMixin(object):
+class ResponseContextMixin(object):
```

Comment:
What is a response context? (Don't reply here, add a docstring)

Reply:
Done.